common/src/main/java/com/wynntils/services/mapdata/attributes/type/MapAttributes.java

```diff
@@ -20,8 +21,12 @@ public interface MapAttributes {
     // 0 means no value specified; inherit
     int getPriority();
 
-    // 0 means no information is available, or suitable for all combat levels
-    int getLevel();
+    // the range of combat levels for which this feature is suitable
+    // If one of the bounds of the range is undefined, Integer.MIN_VALUE or Integer.MAX_VALUE will be used
+    // (eg. Corkus has a minimum level of 80, but no maximum level)
+    // NONE (0-0) or a range of (Integer.MIN_VALUE-Integer.MAX_VALUE) means suitable for all combat levels
+    // null means no information is available
+    RangedValue getLevelRange();
```

Comment:
I see you have answered my question above here already. 

I am not sure I agree with this. First off, max-level. Is there some real-world examples where a map marker would have a max value? I mean, even a level 100 player might go to Ragni from time to time. 

Since players start out at level 1 (right?), we can be tougher here and say that if you mean "suitable for all levels", you should use 1.

That leaves us with two other cases:
a) min level not known (probably will be the likely case for most player created content, like someone sharing a set of their set of personal good but not well-known grinding locations)
b) min level does not apply, like for instance for the province labels ("Gavel", "Wynn", etc)

and one way to express this:
1) set the min level to 0

I think this is okay, that having a min level of 0 leaves some vagueness if the min level is not known, or not applicable.

Reply:
I remember that we discussed this quite a lot, but I don't remember any arguments or what lead you to make this implementation.

Reply:
I went back and re-read the discussion. I think it boiled down to basically you asking for a way to separate level unknown from level-open-for-all or level-does-not-apply, and me saying that it is hard to to, and even if we separate it, it's hard to know how to use it.

I worry that making this complicated just leaves a way for people to make bad data. I would argue that we should just have these possible values:
1: min level of 1, that is, suitable for everyone
>1: only suitable for the recommended min level
0: level unknown or not applicable

and if we want to show only stuff that are below my level, we can have an option "include markers with unknown or n/a level" so people can chose if that should be included or not.

Reply:
And we should ignore the bad data on the Wynncraft map labels, and just set the integer 60 for the string "60-70". So level just becomes a single, trivial integer.

Reply:
Had to change this up a little bit. You forgot to deal with inheritance. I think my version should work for you.